### PR TITLE
Add Solution 2

### PR DESCRIPTION
**Pull request by timothygao8710**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [done] I have tested my code.
- [done] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [done] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/5_Plat/Sweep_Line.mdx [hunk 1]
```diff
@@ -57,8 +57,16 @@ Because each step now runs in linear time and there is a total of $\lceil \log n
...
 
-(Set)
+**Time Complexity:** $\mathcal{O}(n\log n)$
+
+Extending Solution 1, we can use a set instead of divide and conquer. Once again, we define $\delta$ as the shortest distance between two points so far. After sorting the points by x-coordinate, we iterate through them while maintain a running window containing the y-coordinates of all points in $[x-\delta,x]$. 
+
+![Set Solution Sweep Line Bounding Box](./assets/setsolutionboundingbox.png)
+<!--- https://imgur.com/a/iA4cVvk -->
+
+As visit point $P$, we utilize the set to consider all points with y-coordiate in $[P_y-\delta, P_y+\delta]$. The set contains $[P_x-\delta, P_x]$ because of how it is maintained as a running window. Now we have the same bounding box as Solution 1, with at most 6 points inside.
```
Comment (SansPapyrus683) [inline content/5_Plat/Sweep_Line.mdx:67]: > As visit point $P$

?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/5_Plat/Sweep_Line.mdx [hunk 2]
```diff
@@ -57,8 +57,16 @@ Because each step now runs in linear time and there is a total of $\lceil \log n
...
+**Time Complexity:** $\mathcal{O}(n\log n)$
+
+Extending Solution 1, we can use a set instead of divide and conquer. Once again, we define $\delta$ as the shortest distance between two points so far. After sorting the points by x-coordinate, we iterate through them while maintain a running window containing the y-coordinates of all points in $[x-\delta,x]$. 
+
+![Set Solution Sweep Line Bounding Box](./assets/setsolutionboundingbox.png)
+<!--- https://imgur.com/a/iA4cVvk -->
+
+As visit point $P$, we utilize the set to consider all points with y-coordiate in $[P_y-\delta, P_y+\delta]$. The set contains $[P_x-\delta, P_x]$ because of how it is maintained as a running window. Now we have the same bounding box as Solution 1, with at most 6 points inside. 
 
+For each point, we recalculate $\delta = min(\delta, \delta_p)$, and update our set according. Each point is inserted and removed from the set at most once, so the algorithm yields $\mathcal{O}(n\log n)$.
```
Comment (brad182) [inline content/5_Plat/Sweep_Line.mdx:69]:
```suggestion
For each point, we recalculate $\delta = min(\delta, \delta_p)$, and update our set accordingly. Each point is inserted and removed from the set at most once, so the algorithm yields $\mathcal{O}(n\log n)$.
```

**Review by brad182 — APPROVED** [review]
good after addressing small fixes

File: content/5_Plat/Sweep_Line.mdx (hunk 1 above)
Comment (timothygao8710) [reply content/5_Plat/Sweep_Line.mdx]: fixed

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**